provision-contest/ansible/roles/clusterssh/tasks/main.yml

```diff
@@ -22,3 +22,46 @@
     src: clusters.j2
     dest: /home/domjudge/.clusterssh/clusters
     mode: 0644
+
+- name: Install tmux-cssh
+  when: not WF_RESTRICTED_NETWORK
+  get_url:
+    url: https://raw.githubusercontent.com/peikk0/tmux-cssh/master/tmux-cssh
+    dest: /usr/local/bin/tmux-cssh
+    mode: 0755
+    owner: root
+    group: root
+
+- name: Install tmux-cssh
+  when: WF_RESTRICTED_NETWORK
+  copy:
+    src: tmux-cssh
+    dest: /usr/local/bin/tmux-cssh
+    mode: 0755
+    owner: root
+    group: root
+
+- name: Set tmux-cssh shorthands
+  template:
+    owner: root
+    group: root
+    mode: 0755
+    src: tmux-cluster.sh.j2
+    dest: /usr/local/bin/{{ item.short }}
+  loop:
+    - short: ssh-js
```

Comment:
I prefer `jh` but if it is just me I can alias it 😀

Reply:
I'm fine with any comment, so feel free to push the right naming afterwards. I already changed the `js` to `jh` as it does indeed make more sense.

For the group take a look at the hosts file where subgroups do matter (so the group `domserver` does contain everything in the group and the hosts from its children)